src/shared/api/voucherApi.ts

```
@@ -0,0 +1,275 @@
+	data: {
+		uploadUrl: string;
+		key: string;
+		expiresAt: string;
+	};
+	code: number;
+}
+
+/**
+ * API 응답이 유효한지 검증합니다.
+ */
+const isValidVoucherIssuerResponse = (response: any): response is VoucherIssuerApiResponse => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)